src/BitmapText.cpp

```diff
@@ -711,7 +712,8 @@ void BitmapText::DrawPrimitives()
 		// render the diffuse pass
 		if( m_bRainbowScroll )
 		{
-			int color_index = int(RageTimer::GetTimeSinceStartFast() / 0.200) % RAINBOW_COLORS.size();
+			int iSeed = GetRandomInt();
+			int color_index = std::abs(iSeed);
```

Comment:
Why do we need `abs` if iSeed is non-negative?

Reply:
Also we need to` % RAINBOW_COLORS.size();` otherwise the index is likely out of bounds.

Reply:
Ah true. I had forgot I made my function unable to return a negative integer. Lol